core/transaction/validator.go

```diff
@@ -178,6 +181,12 @@ func VerifyTransactionWithLedger(Tx *Transaction) ErrCode {
 	return ErrNoError
 }
 
+// CheckInputOutputLen validate len(inputs/outputs) less than MAX of ReferTxOutputIndex
+func CheckInputOutputTooLong(tx *Transaction) bool {
+	// FIXME: assume count of Inputs in a Transaction never over 2147483648
```

Comment:
I think we are good since len() returns a int type anyway

Reply:
2147483648 will be regarded as negative on 32 bits platform. And int of golang will be either 32 or 64 bits depend on implementation-specific.
I am not handle this corner case since transaction contain so much Inputs/Outputs would be a huge msg can't broadcast on network at all.

Reply:
I mean that len(tx.Inputs) will return a negative cause bool ( len(tx.Inputs) > 65535 ) as false.